BoldersRebornPrefs/Resources/Landscape.plist

```diff
@@ -8,7 +8,7 @@
 | Landscape support is not planned at the moment.                  |
 |==================================================================|
 */
-
+-->
```

Comment:
I added "-->" so I can compile the package otherwise I will get this error

BoldersReborn/.theos/_/Library/PreferenceBundles/BoldersRebornPrefs.bundle/Landscape.plist: Property List error: Unterminated comment started on line 2 / JSON error: JSON text did not start with array or object and option to allow fragments not set. around line 1, column 0.
make: *** [internal-package] Error 1